doc/cascadia/profiles.schema.json

```diff
@@ -1356,44 +1404,6 @@
           "description": "Use to set a path to a pixel shader to use with the Terminal. Overrides `experimental.retroTerminalEffect`. This is an experimental feature, and its continued existence is not guaranteed.",
           "type": "string"
         },
-        "fontFace": {
-          "default": "Cascadia Mono",
-          "description": "Name of the font face used in the profile.",
-          "type": "string"
-        },
-        "fontSize": {
-          "default": 12,
-          "description": "Size of the font in points.",
-          "minimum": 1,
-          "type": "integer"
-        },
-        "fontWeight": {
-          "default": "normal",
-          "description": "Sets the weight (lightness or heaviness of the strokes) for the given font. Possible values:\n -\"thin\"\n -\"extra-light\"\n -\"light\"\n -\"semi-light\"\n -\"normal\" (default)\n -\"medium\"\n -\"semi-bold\"\n -\"bold\"\n -\"extra-bold\"\n -\"black\"\n -\"extra-black\"\n or the corresponding numeric representation of OpenType font weight.",
-          "oneOf": [
-            {
-              "enum": [
-                "thin",
-                "extra-light",
-                "light",
-                "semi-light",
-                "normal",
-                "medium",
-                "semi-bold",
-                "bold",
-                "extra-bold",
-                "black",
-                "extra-black"
-              ],
-              "type": "string"
-            },
-            {
-              "maximum": 990,
-              "minimum": 100,
-              "type": "integer"
-            }
-          ]
-        },
```

Comment:
nit: you can probably keep these and add the `"deprecated"` property (and add "[deprecated]" to the description)

Reply:
Good call! Made the change